cmd/migrations/main.go

```diff
@@ -24,33 +24,36 @@ func main() {
 	config, err := cfg.GetConfig()
 	if err != nil {
 		log.Printf("Failed to get config from .env: %+v", err)
+
 		return
 	}
 
 	logger, err := lgr.New(config.LogLevel)
 	if err != nil {
 		log.Printf("failed to create logger: %v", err)
+
 		return
 	}
 
 	db, err := pg.ConnectDB(config.DB)
 	if err != nil {
 		logger.Errorf("%+v", err)
+
 		return
 	}
 
 	direction := flag.String("migrate", "", "applying migrations 'up/down'")
 	flag.Parse()
 
 	if *direction != up && *direction != down {
-		fmt.Println("Wrong flag provided, choose '-migrate up' or '-migrate down'")
+		log.Println("Wrong flag provided, choose '-migrate up' or '-migrate down'")
+
```

Comment:
I think we should use the `up` and `down` constants in all the strings here.

Reply:
fix